pandas/core/tools/datetimes.py

```diff
@@ -853,6 +853,17 @@ def to_datetime(
     ...                origin=pd.Timestamp('1960-01-01'))
     DatetimeIndex(['1960-01-02', '1960-01-03', '1960-01-04'], \
 dtype='datetime64[ns]', freq=None)
+
```

Comment:
can you fix the one right above as well, thanks.

Reply:
Sure. Have fixed the example above.